src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/SchemaRepository.cs

```diff
@@ -24,12 +24,17 @@ public void RegisterType(Type type, string schemaId)
             {
                 var conflictingType = _reservedIds.First(entry => entry.Value == schemaId).Key;
 
-                throw new InvalidOperationException(
-                    $"Can't use schemaId \"${schemaId}\" for type \"${type}\". " +
-                    $"The same schemaId is already used for type \"${conflictingType}\"");
+                if(conflictingType != type)
```

Comment:
```suggestion
                if (conflictingType != type)
```

Reply:
@martincostello ok, what's the best course of action ? Do i create a new WebSites with only the problematic endpoint (that should not crash anymore) ? Do i add a new endpoint / controller (in that case in which WebSites, Projects ? )

Thanks for your help.

Reply:
The lowest-effort approach (hopefully) should be to add to one of the existing SwaggerGen tests that declares an endpoint in code and then checks the behaviour of what was generated. If you add a test that throws an exception without your change, then passes after it's made with the correct eventual behaviour should make us happy from a regression testing point of view 😄

I would consider adding a new project a "last resort" as they're relatively heavyweight and are typically used for end-to-end scenarios, rather than testing specific scenarios.